internal/models/device.go

```diff
@@ -12,6 +12,7 @@ type Device struct {
 	PublicKey string      `gorm:"uniqueIndex" json:"public_key"`
 	Peers     []*Peer     `json:"-"`
 	PeerList  []uuid.UUID `gorm:"-" json:"peers" example:"97d5214a-8c51-4772-b492-53de034740c5"`
+	Hostname  string      `json:"hostname"`
```

Comment:
Please also add `example:"fedora"` so we get a pretty representation in the API docs